client/extensions/woocommerce/app/dashboard/setup-tasks.js

```diff
@@ -112,7 +112,7 @@ class SetupTasks extends Component {
 		return [
 			{
 				checked: hasProducts,
-				explanation: translate( 'Start by adding the first product to your store.' ),
+				explanation: translate( 'Start by adding the first product to your\u00a0store.' ),
```

Comment:
Its the unicode character for a non breaking space. Gotta prevent those orphans!